sandy/core/cov.py

```diff
@@ -2327,13 +2385,42 @@ def sample_distribution(dim, nsmp, seed=None, pdf='normal'):
     >>> sandy.cov.sample_distribution(2, 3, seed=11)
     array([[ 1.74945474, -0.286073  , -0.48456513],
            [-2.65331856, -0.00828463, -0.31963136]])
+
+    >>> sandy.cov.sample_distribution(2, 3, seed=11, pdf='lognormal')
+    array([[ 2.03418551, -0.44275205, -0.5276337 ],
+           [-0.92235485, -0.29775364, -0.45810561]])
+
+    >>> sandy.cov.sample_distribution(2, 3, seed=11, pdf='uniform')
+    array([[-1.10757829, -1.66458659, -0.12741476],
+           [ 0.77919399, -0.27642282, -0.05048201]])
+
     >>> sandy.cov.sample_distribution(2, 1000000, seed=11).mean().round(5)
     0.00025
+
+    >>> sandy.cov.sample_distribution(2, 1000000, seed=11, pdf='lognormal').mean().round(5)
+    -0.00047
+
+    >>> sandy.cov.sample_distribution(2, 1000000, seed=11, pdf='uniform').mean().round(5)
+    -0.00115
+
+    >>> sandy.cov.sample_distribution(2, 1000000, seed=11, mean=3).mean().round(5)
+    3.00025
+
+    >>> sandy.cov.sample_distribution(2, 1000000, seed=11, mean=3).std().round(5)
+    0.99919
     """
     np.random.seed(seed=seed)
     if pdf == 'normal':
         y = np.random.randn(dim, nsmp)
-    return y
+    elif pdf == 'lognormal':

```

Comment:
can we get a visual test on a separate notebook submitted to https://luca-fiorito-11.github.io/sandy_notebooks/